dev/io.openliberty.io.smallrye.reactive.messaging-provider4/bnd.bnd

```diff
@@ -43,13 +43,16 @@ IBM-Default-Config: OSGI-INF/wlp/defaultInstances.xml
 Private-Package: \
 	io.smallrye.reactive.messaging.providers.connectors;-split-package:=merge-first,\
 	io.smallrye.reactive.messaging.*,\
-	io.openliberty.microprofile.reactive.messaging30.internal
+	io.openliberty.microprofile.reactive.messaging30.internal,\
+	io.openliberty.microprofile.reactive.messaging30.resources;version=1.0
```

Comment:
1. There's no need to version packages that aren't exported
2. This package also has a `package-info.java` with a version defined. You can either put the version here in the bnd, or in the `package-info.java`, you don't want it in both because they can then get out of sync.